schutzbot/manifest_tests.sh

```diff
@@ -1,10 +1,37 @@
 #!/bin/bash
+function nvrGreaterOrEqual {
+    local rpm_name=$1
+    local min_version=$2
+
+    set +e
+
+    rpm_version=$(rpm -q --qf "%{version}" "${rpm_name}")
+    rpmdev-vercmp "${rpm_version}" "${min_version}" 1>&2
+    if [ "$?" != "12" ]; then
+        # 0 - rpm_version == min_version
+        # 11 - rpm_version > min_version
+        # 12 - rpm_version < min_version
+        echo "DEBUG: ${rpm_version} >= ${min_version}" 1>&2
+        set -e
+        return
+    fi
+
+    set -e
+    false
+}
+
 set -euxo pipefail
 # Get OS details.
 source /etc/os-release
 ARCH=$(uname -m)
-DISTRO_CODE="${DISTRO_CODE:-${ID}-${VERSION_ID//./}}"
-
+# Since rhel-9.3 and rhel-8.9 the manifests are stored with dotted notation (rhel-9.3 instead of rhel-93, hence
+# adding this to handle the new notation and ensure backwards compatibility
+COMPARE=$(nvrGreaterOrEqual "osbuild-composer" "100")
+if $COMPARE -eq 0 || $COMPARE -eq 11; then
```

Comment:
this condition is incorrect. The logic is already inside the helper function. It should be used as:

```
if nvrGreaterOrEqual "osbuild-composer" "100"; then
```


see the osbuild-composer test suite for example.